### ISSUE TITLE
Apple VoiceOver does not read all Options in Listbox

### ISSUE THREAD

**Issue by stianmartinsen** [tailwindlabs/headlessui#947]
**What package within Headless UI are you using?**

@headlessui/react

**What version of that package are you using?**

v1.4.2

**What browser are you using?**

Safari

**Reproduction URL**

https://headlessui.dev/react/listbox

**Describe your issue**

When using Apple's VoiceOver in Safari, not all options in the Listbox component are read out loud when navigating using the arrow keys. Only the selected option, and the ones preceding it, works. When navigating to any options _after_ the selected option, VoiceOver doesn't say anything.

<img width="793" alt="Skjermbilde 2021-11-22 kl  10 47 48" src="https://user-images.githubusercontent.com/2544063/142840040-293184cd-83fe-4424-8714-9bb7552d7997.png">

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

As @thecrypticace noted this is a browser bug / VoiceOver bug. We can switch to the roving tab index strategy instead but that's a big effort and doesn't fix potential future browser/VO bugs either.

Going to close the issue for now, and if this bug still exists after some time has passed, then we can reassess and implement the roving tab index strategy instead.

**Comment by sebammon** [comment]
@RobinMalfait @thecrypticace we currently have users affected by this issue. LMK if we can re-open this issue or if I should create a new issue?

I realise that you're a small team, so LMK if there is any way that I can contribute / help with the issue.
